portfolio/pom.xml

```diff
@@ -17,12 +17,33 @@
   </properties>
 
   <dependencies>
+
```

Comment:
The "portfolio" directory is a little confusing.  Could you just move everything to the root directory?